src/asmcnc/comms/router_machine.py

```diff
@@ -169,11 +169,17 @@ def __init__(self, win_serial_port, screen_manager, settings_manager, localizati
     def reconnect_serial_connection(self):
         if self.s.is_connected():
```

Comment:
First 5 lines of this function are the same as close_serial_connection, could just call close_serial_connection instead to avoid duplicate code?

Reply:
👍